implementations/rust/ockam/ockam_api/src/cli_state.rs

```diff
@@ -130,9 +131,9 @@ impl CliState {
 
     /// Returns the default directory for the CLI state.
     pub fn default_dir() -> Result<PathBuf> {
-        Ok(match std::env::var("OCKAM_HOME") {
-            Ok(dir) => PathBuf::from(&dir),
-            Err(_) => dirs::home_dir()
+        Ok(match get_env::<String>("OCKAM_HOME") {
+            Ok(Some(dir)) => PathBuf::from(&dir),
```

Comment:
Instead of going `PathBuf::from` here we can use a `FromString` instance and provide
```
dirs::home_dir().ok_or_else(|| CliStateError::NotFound("home dir".to_string()))?.join(".ockam")
```
as a default value